site/NotFoundPage.tsx

```diff
@@ -1,48 +1,39 @@
-import React from "react"
+import * as React from "react"
 import { Head } from "./Head.js"
 import { SiteHeader } from "./SiteHeader.js"
 import { SiteFooter } from "./SiteFooter.js"
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome/index.js"
-import { faSearch } from "@fortawesome/free-solid-svg-icons"
 import { Html } from "./Html.js"
+import NotFoundPageForm from "./NotFoundPageForm.js"
 
-export const NotFoundPage = (props: { baseUrl: string }) => {
+export default function NotFoundPage({ baseUrl }: { baseUrl: string }) {
     return (
         <Html>
             <Head
-                canonicalUrl={`${props.baseUrl}/search`}
+                canonicalUrl={`${baseUrl}/search`}
                 pageTitle="404 Not Found"
                 pageDesc="Search articles and charts on Our World in Data."
-                baseUrl={props.baseUrl}
+                baseUrl={baseUrl}
             />
             <body className="NotFoundPage">
-                <SiteHeader baseUrl={props.baseUrl} />
+                <SiteHeader baseUrl={baseUrl} />
                 <main>
-                    <h1>Sorry, that page doesn’t exist!</h1>
-                    <p>
-                        You can search below or{" "}
-                        <a href="/">return to the homepage</a>.
+                    <NotFoundPageIcon />
+                    <h1 className="NotFoundPage__heading subtitle-1">
+                        Sorry, that page doesn’t exist!
+                    </h1>
+                    <p className="body-2-semibold">
+                        You may have followed an outdated link or have mistyped
+                        the URL.
                     </p>
-                    <form action="/search" method="GET">
-                        <div className="inputWrapper">
-                            <input
-                                id="search_q"
-                                type="search"
-                                name="q"
-                                autoFocus
-                            />
-                            <FontAwesomeIcon icon={faSearch} />
-                        </div>
-                        <button
-                            aria-label="Submit search"
-                            className="btn"
-                            type="submit"
-                        >
-                            Search
-                        </button>
-                    </form>
+                    <p className="body-2-semibold">
+                        You can search for what you were hoping to find below or{" "}
+                        <a href="/">visit our homepage</a>.
```

Comment:
This link is missing a hover style - is it meant to be `owid-link-90` ? (we should probably add a util class for that mixin, like we do for typography)

Reply:
Almost, it should be blue-60 color. I'll still use the mixin.